frontend/src/components/phase-banner/index.js

```diff
@@ -1,26 +1,26 @@
-import React from 'react'
+/** @jsx jsx */
 import PropTypes from 'prop-types'
-import styled from '@emotion/styled'
+import { jsx, css } from '@emotion/core'
 import { Text } from '../text'
-import tag from 'clean-tag'
 
-export const PhaseBannerContainer = styled(tag.div)`
-  margin: 0;
-  padding-bottom: 10px;
-  border-bottom: 1px solid black;
-`
-
-export const PhaseBanner = ({ phase, message, phaseColor, ...rest }) => (
-  <PhaseBannerContainer>
+export const PhaseBanner = ({ phase, children, phaseColor, ...rest }) => (
+  <div
+    css={css`
+      margin: 0;
+      padding-bottom: 10px;
+      border-bottom: 1px solid black;
+    `}
+    {...rest}
```

Comment:
I think, as per our conversation, applying the rest of the props may not be necessary? Up to you! Otherwise looks good to me!